### PR TITLE
Controllers2

### PR DESCRIPTION
**Pull request by DavidUKr**
implemented all use cases without "Get mentions"
changes to model structure: Posting->Post->Reply
now only 3 controllers
+advice package

### CODE REVIEW

File: src/main/java/app/Twiter/control/UserController.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+    //CREATE
+    @PutMapping(consumes = MediaType.APPLICATION_JSON_VALUE)
+    public void registerUser(@RequestBody User user){
+        userService.registerUser(user);
+    }
+
+    //READ
+    @Autowired
+    private UserService userService;
```
Comment (dmolnar-wb) [inline src/main/java/app/Twiter/control/UserController.java:23]: Please declare your variables/constants at the beginning of the class, before the methods.

**Comment by dmolnar-wb** [comment]
.

**Comment by dmolnar-wb** [comment]
You have too many projects in this Repo...please keep only the Twitter App in this Repo. Thanks!

File: src/main/java/app/Twiter/control/FeedController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
+package app.Twiter.control;
```
Comment (dmolnar-wb) [inline src/main/java/app/Twiter/control/FeedController.java:1]: This package should be called controller

File: src/main/java/app/Twiter/control/UserController.java (hunk 1 above)
Comment (DavidUKr) [reply src/main/java/app/Twiter/control/UserController.java:23]: Thank you!! Left it there by mistake

File: src/main/java/app/Twiter/control/FeedController.java (hunk 2 above)
Comment (DavidUKr) [reply src/main/java/app/Twiter/control/FeedController.java:1]: Thank you!

**Pull request merged**